tasks/main.yml

```diff
@@ -1,12 +1,9 @@
-#####################
-
 - name: "Install util-linux"
-  apt:
+  package:
```

Comment:
`package` is supported only from Ansible 2.0.0, so it will be a new major version.